testing/asserts_test.ts

```diff
@@ -40,11 +41,14 @@ Deno.test("testingEqualDifferentZero", () => {
 Deno.test("testingEqual", function (): void {
   assert(equal("world", "world"));
   assert(!equal("hello", "world"));
+  assertFalse(equal("hello", "world"));
```

Comment:
All the tests follow the format `assertFalse(equal(/* --snpi */))`.

Is this not the same as `assertNotEquals` but the output is less helpful for debugging?

IMHO we should lead by example and write tests in an idiomatic way wherever possible. I think these tests are not idiomatic because they are effectively testing the `equals` function which is already covered.

Could we instead write tests which assert different falsy values like `assertFalse(false)` and `assertFalse(null)` etc?

Additionally, it would be nice to assert that this assertion will fail when passed truthy values.

Reply:
I just add these tests, I didn't remove the ones related to `equals` but if that's wanted I can do it too.

Reply:
Unless you think they add significant value over the new tests you added (which look great btw 😊) I would lean towards removing them.

Reason being that if we add lots of tests which are not needed then the tests will run slower. Of course, these tests wont make a significant difference on their own, but if every new feature adds 5 unnecessary tests then the test suite quickly becomes bloated.

What do you think @UltiRequiem ?